python-package/lightgbm/sklearn.py

```diff
@@ -606,11 +610,13 @@ def feature_importances_(self):
         Note
         ----
         Feature importance in sklearn interface used to normalize to 1,
-        it's deprecated after 2.0.4 and same as Booster.feature_importance() now.
+        it's deprecated after 2.0.4 and is the same as Booster.feature_importance() now.
+        ``importance_type`` attribute is passed to the function to configure the 
```

Comment:
Please add some explanation from where `importance_type` appears.

``importance_type`` attribute **specified in the constructor** is passed